content/blogposts/2024/announcing-prompt-library.md

```diff
@@ -0,0 +1,149 @@
+
+## 2. “Cody is not following my organization’s style guide”
+
+Suppose your organization has a specific Python coding style guide that mandates the use of snake_case for variable names and requires docstrings for all functions. However, Cody might generate code that doesn't adhere to these conventions, leading to inconsistencies in your codebase.
+
+```python
+def processData(inputData):
+    """Process the input data and return the result."""
+    processedData = inputData * 2  # This should be snake_case
+    return processedData
+```
+
+To solve this, you can create a comprehensive prompt that instructs Cody to follow your organization's style guide:
+
+"Generate Python code that adheres to our organization's style guide, which includes the following requirements:
+
+- Use snake_case for all variable and function names.
+- Include docstrings for all functions, describing their purpose and parameters.
+- Follow PEP 8 guidelines for code formatting, including indentation and line length."
+
+This prompt can be shared with your colleagues to ensure consistent code style across the team.
+
+## 3. “Cody is not generating tests in the right format”
+
+Imagine you are working with Golang and need Cody to generate unit tests that comply with your team’s conventions. Your organization follows a specific structure for tests, including the use of table-driven tests and `require` from the `testify` package for assertions.
+
+However, Cody might provide a simple test that does not align with your standards:
+
+```go
+func TestAdd(t *testing.T) {
+    result := Add(2, 3)
+    if result != 5 {
+        t.Errorf("expected 5, got %d", result)
+    }
+}
+
+```
+
+To ensure consistency, create a prompt in the Prompt Library specifying your test requirements:
+
+"Please generate Golang unit tests that adhere to the following conventions:
+
+- Use table-driven tests for different input cases.
+- Use the `require` package from `testify` for assertions.
+- Follow idiomatic Go test naming conventions and structure."
+
+When this prompt is used, Cody will generate tests like the following:
+
+```go
+func TestAdd(t *testing.T) {
+    testCases := []struct {
+        name     string
+        a, b     int
+        expected int
+    }{
+        {"add positive numbers", 2, 3, 5},
+        {"add zero", 0, 5, 5},
+        {"add negative numbers", -2, -3, -5},
+    }
+
+    for _, tc := range testCases {
+        t.Run(tc.name, func(t *testing.T) {
+            result := Add(tc.a, tc.b)
+            require.Equal(t, tc.expected, result)
+        })
+    }
+}
+
+```
+
+By saving and sharing this prompt within your team, you ensure that Cody consistently generates tests in the correct format, saving time and reducing review cycles.
+
+# Learn more
+
+You can [learn more about this new feature on our docs.](https://sourcegraph.com/docs/cody/capabilities/commands)
```

Comment:
I would add a few sentences about how to get started with prompts today and a shortlink to the prompt library. something like "Prompt Library is available starting today. Get started by heading over to ..." also include a CTA for people that aren't using Cody yet to download and try Cody and the prompt library, and then finish up with the docs link. :)